_rules/audio-as-media-alternative-afb423.md

```diff
@@ -58,16 +59,18 @@ There are no major accessibility support issues known for this rule.
 An audio element that describes some of the text on the same page. The text on the page labels the audio as an alternative.
 
 ```html
-<p>
-	A part of a speech by John F. Kennedy: We choose to go to the moon in this decade and do the other things, not because
-	they are easy, but because they are hard, because that goal will serve to organize and measure the best of our
-	energies and skills, because that challenge is one that we are willing to accept, one we are unwilling to postpone,
-	and one which we intend to win, and the others, too.
-</p>
-<p>
-	You can also listen to the audio file below to hear the above part of the speech.
-</p>
-<audio src="/test-assets/moon-audio/moon-speech.mp3" controls></audio>
+<section lang="en">
```

Comment:
Can somebody explain to me why we are adding this in the first place? If the example doesn't include the HTML tag, then aren't we simply assuming it is there and has a correct lang attribute?

Reply:
@ShadowBB this is the first point in https://github.com/act-rules/act-rules.github.io/issues/671#issuecomment-530309607

As far as I understand, the code of each example is evaluated in isolation and the evaluator (whether tool or human) needs a way to know the actual language of the example in order to evaluate it (I can't evaluate whether "is" is descriptive without knowing if it is supposed to be an English word or a Danish one ("is" in Danish = "ice" in English)).

Since the code of an example is all that an evaluator has, the code must be self-contained for everything needed for the evaluation.

Reply:
Thank you for the explanation!